components/server/src/session-handler.ts

```diff
@@ -40,7 +42,25 @@ export class SessionHandlerProvider {
 
         options.store = this.createStore();
 
-        this.sessionHandler = session(options);
+        this.sessionHandler = async (req: express.Request, res: express.Response) => {
+            const user = req.user;
```

Comment:
User is not set, here, yet. It can't, because we don't have a session.

What about making this a "global" feature flag?

Reply:
If the user ends up being unset, which is fine (and we can remove the code) then it would already effectively play as a global one (as user is optional)

Reply:
That's what I meant, yes. :+1:

Reply:
Should be fixed in latest version, just waiting (forever) for the preview